CMakeLists.txt

```diff
@@ -3,8 +3,7 @@
 
 cmake_minimum_required(VERSION 3.5)
 
-# Specify search path for CMake modules to be loaded by include()
-# and find_package()
```

Comment:
Yannis: It seems that these two lines were manually merged into one line. When I ran cmake-format on the original CMakeLists.txt, it did not change these two lines.

This doesn't really matter. I am just testing cmake-format.